README.md

```diff
@@ -19,105 +18,137 @@ Or install from package managers
+              "type": "string",
+              "enum": [
+                "off-topic",
+                "too heated",
+                "resolved",
+                "spam"
+              ]
+            }
+          }
+        }
+      }
+    }
+  },
+  "responses": {
+    "204": {
+      "description": "Empty response"
+    }
+  },
+  "x-code-samples": [
+    {
+      "lang": "Shell",
+      "source": "curl \\\n  -XPUT \\\n  -H\"Accept: application/vnd.github.v3+json\" \\\n  https://developer.github.com/repos/octocat/:repo/issues/:issue_number/lock"
+    },
     {
-      "locked": true,
-      "active_lock_reason": "too heated"
+      "lang": "JS",
+      "source": "octokit.issues.get({\n  owner: 'octocat',\n  repo: 'hello-world',\n  issue_number: 1\n})"
     }
   ],
-  "description": "Users with push access can lock an issue or pull request's conversation.\n\nNote that, if you choose not to pass any parameters, you'll need to set `Content-Length` to zero when calling out to this endpoint. For more information, see \"[HTTP verbs](https://developer.github.com/v3/#http-verbs).\"",
-  "responses": [
+  "x-github": {
+    "legacy": false,
+    "enabledForApps": true,
+    "githubCloudOnly": false
+  },
+  "x-changes": [
     {
-      "headers": {
-        "status": "204 No Content",
-        "content-type": "application/json; charset=utf-8"
+      "type": "parameter",
+      "date": "2019-04-10",
+      "note": "\"number\" parameter renamed to \"issue_number\"",
+      "meta": {
+        "before": "number",
+        "after": "issue_number"
       }
     }
-  ],
-  "idName": "lock",
-  "documentationUrl": "https://developer.github.com/v3/issues/#lock-an-issue"
+  ]
 }
 ````
 
-Both endpoints or parameters may be deprecated. The `date` timestamp can be used to determine how long an Octokit library wants to support the endpoint / parameter.
+Both endpoints or parameters may be deprecated. The `date` timestamp can be used to determine how long an Octokit library wants to support the endpoint / parameter. Deprecation information is located in the `x-changes` array of an operation.
 
 Example for a deprecated parameter
 
 ```json
 {
-  "name": "number",
-  "type": "integer",
-  "required": false,
-  "description": "",
-  "location": "url",
-  "deprecated": {
-    "date": "2019-04-10",
-    "message": "\"number\" parameter renamed to \"issue_number\"",
-    "before": {
-      "idName": "number"
-    },
-    "after": {
-      "idName": "issue_number"
-    }
+  "type": "parameter",
+  "date": "2019-04-10",
+  "note": "\"number\" parameter renamed to \"issue_number\"",
+  "meta": {
+    "before": "number",
+    "after": "issue_number"
   }
 }
 ```
 
-Deprecated endpoints have a "deprecated" property
+Deprecated endpoints have a type of "idName"
```

Comment:
There are deprecations for parameter names, e.g. https://github.com/octokit/routes/blob/35d0f2e93089004bbb1e8daec3af51b89b9f76b7/openapi/api.github.com/operations/issues/list-labels-on-issue.json#L95-L105

Maybe we can rephrase it to something like 

> Changes such as deprecations are listed in the `x-changes` key?

And then show an example including the `"x-changes": [ ... ]`?

Reply:
i have something similar in there with two `x-changes` examples...the first one is a parameter change...this one is second one...it's not as easy to see that in the diff tho...this is what's in the readme now:

---

... Deprecation information is located in the x-changes array of an operation.

Example for a deprecated parameter
```
{
  ...
}
```

Deprecated endpoints have a type of "idName"
```
{
  ...
}
```

---

let me know if this still feels wrong/off